lib/sage/exceptions.ex

```diff
@@ -96,12 +96,12 @@ defmodule Sage.MalformedTransactionReturnError do
   @moduledoc """
   Raised at runtime when the transaction or operation has an malformed return.
   """
-  defexception [:transaction, :return]
+  defexception [:name, :transaction, :return]
```

Comment:
@superhawk610 can we please use `stage` instead of `name` everywhere? I think it would be much more clear this way :). Other than that, LGTM